yb_migrate/src/migration/common.go

```diff
@@ -282,7 +282,7 @@ func SelectVersionQuery(dbType string, dbConnStr string) string {
 		}
 		defer db.Close()
 
-		query := "SELECT VERSION FROM V$INSTANCE"
+		query := "SELECT BANNER FROM V$VERSION"
```

Comment:
Add a comment showing sample output of the query. So no one assumes that it is just a numeric version.
e.g.

```
// Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production
```